src/FidoInterface.cpp

```diff
@@ -144,7 +144,7 @@ double FidoInterface::estimatePriors(Scores& peptideScores) {
 	      tmp_prior = (tmp_prior * prior_protein * (size - index)) / (index + 1);
 	      prior +=  pow(-1.0,(int)index) * tmp_prior;
       }
-      /* update computed prior */
+      //update computed prior
```

Comment:
It's generally better to avoid making changes like this, which are just a matter of personal taste. Such changes are fine if you're already touching the code for other reasons, but this sort of thing just creates noise in the revision history.

Reply:
reverting back